### PR TITLE
Replace pidof with pgrep

### PR DESCRIPTION
**Pull request by adriansuarez**
Replace pidof with pgrep and improve bare-metal instructions.

### CODE REVIEW

File: nuocd/nuodb_threads.py [hunk 1]
```diff
@@ -36,9 +36,8 @@ def signal_handler(sig, frame):
     _pids = None
     try:
         # assume only one process by given name
-        _pid = subprocess.check_output(["pidof", process_name])
-        # _pid = subprocess.check_output(["pgrep", process_name ])
-        _pids = _pid[:-1].split()
```
Comment (adriansuarez) [inline nuocd/nuodb_threads.py:41]: I think this was stripping away the trailing newline, but `str.split()` already does that.

File: README.md [hunk 2]
```diff
@@ -190,9 +192,8 @@ Replace the `<hostinflux>` placeholder with the hostname for a running InfluxDB
 ```
 sudo cp conf/nuodb.conf /etc/telegraf/telegraf.d
 sudo cp conf/outputs.conf /etc/telegraf/telegraf.d
-sudo edit /etc/telegraf/telegraf.conf
```
Comment (adriansuarez) [inline README.md:193]: I'm not sure what this command is for, but I didn't seem to need it. This is supposed to interactively edit the file, but I don't see what change we need to make to it.

**Review by vegichan — APPROVED** [review]
looks better to me!

**Comment by butson** [comment]
Pgrep will have problems if other processes are running that contain nuodb in name.  You should include ^nuodb$.  Pidof is more reliable

**Pull request merged**